lib/src/span.dart

```diff
@@ -106,3 +108,74 @@ class SourceSpanBase extends SourceSpanMixin {
     }
   }
 }
+
+// Consider moving these to instance methods in the next breaking release, so
```

Comment:
Can you file an issue for this and remove the comment? Or file an issue and change this to a // TODO referring to the issue.

As a comment this is very likely to get lost.

Reply:
Done.